simp-testing/README.md

```diff
@@ -12,73 +12,178 @@
+  - The default passwords for everything (users, LDAP) is P@ssw0rdP@ssw0rd.
+    You change it in vars.json with new_password.
+  - Changing the LDAP Password doesn't work at this time.
+  - It only works for systems with LDAP set up.  It will change
+    the basedn to match what you have in packer.yaml for DOMAIN or use
+    the default.  It sets up user1, user2, admin1, admin2.
+  - The distribution DVD is looked for in the /net/ISO/Distribution_ISOs directory.
+    It is hard coded in simp.json.template right now.
 
-2\. Run the test: Run the script:
-     simp-packer_test.sh <full path to test directory from #1>
-
-It will run and create an output file beginning with the date in the test directory.
-All processing is in a temporary working directory created under the test directory.
-The script will replace environment variables in the template script and set up and export
-environment variables.  It also replaces some variables in the json script and removes
-any comments.
-
-3\. Once the environment is set up, packer uses the vars.json file to find the ISO you want to
-use.  The vars file also contains the password to be used for the simp login.
-
-packer installs the iso, then packer uses the simp.json file to step it through
-configuring simp according to the simp_conf.yaml and running simp bootstrap.
-This is done in the build section and the start of the provisioning section.
-
-Once the setup is complete, the tests are run.  These are the final sections in the
-provisioning section of the simp.json.template.  Currently
-it tests the following:
-1) That the build of the puppet server is successful.
-2) Puppet server is up and running and listening on the ports configured in simp_conf.yaml
-3) It verifies that FIPS is setting match across the simp_conf.yaml, the simp_def.yaml and
-   in the operational environment.
-4) Checks that selinux matches the simp_conf.yaml selinux::secure setting.
-5) If simp_crypt_disk is used in the simp.conf, it verifies that the disk is encrypted.
-
-### NOTES: 
-The templates directory:
-  The templates directory was created because environment variables could not be passed to
-scripts that were run under sudo, even using the -E option.  So normally all environment variables
-are set and passed to packer using the variables section and then individually passed to scripts
-running on the system using the local variable section for the script. This was not working for scripts
-run using the sudo command.  To get around this and have one place to declare variables (packer.profile)
-the start up scripts replaces the variables in the template/scripts with the value and copies the hardcoded
-script to the working environment.  In the template the $ is dropped from in from of the variable. 
+
+You don't have to set anything in the packer.yaml file on CentOS 7.  For CentOS 6
+you need to change the NAT_INTERFACE and HOST_ONLY_INTERFACE to eth0 and eth1
+respectively.
 
 
+2\. Run the test: Run the script:
```

Comment:
You need to copy simp-testing to whatever dir you want to run simp_packer_test.sh in